src/loki/chart/values.yaml

```diff
@@ -1 +0,0 @@
-scalable: false
```

Comment:
Should we still have an empty default values file? Thought helm would blow up somewhere without it, but apparently it kinda works 🤔